.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and publish scroll-alpine image
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Clone, Build, Publish
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Update submodules recursively
+        run: git submodule update --init --recursive
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: '21'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Dockerhub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build deploy image
+        id: build_deploy_image
+        env:
+          REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          REPOSITORY: scrolltech/scroll-stack-contracts
+        uses: docker/build-push-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
+          push: true
+          context: .
+          file: docker/Dockerfile.deploy
+          tags: |
+            ${{ env.REPOSITORY }}:deploy-${{ github.ref_name }}
```

Comment:
Can we use commit digest instead, as the tag? (Prefix of `github.sha`)

Reply:
not sure about this, as it is used on scroll-sdk, it will make it less clear which image is the newest. I find the semver tagging clearer.
Maybe we can get @dghelm opinion's on this

Reply:
approved by daniel.